guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md

```diff
@@ -23,4 +23,4 @@ During the installation, a  message similar to the following displays:
 ### Solution
 
 There is a known issue with <code>xdebug</code> that can affect Magento installations or access to the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} or {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} after installation.</p>
-<p>For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html#known-devbeta-xdebug).
+<p>For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html).
```

Comment:
```suggestion

For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html).
```

Reply:
No, that's out of scope for this PR.

Reply:
Actually, never mind, I'll do it real quick.

Reply:
you can just accept the suggested change right on GitHub

Reply:
If a line starts from HTML tag, Markdown would left unparsed. The similar issue like with Markdown in HTML tables.